rainier-plot/src/main/scala/com/stripe/rainier/plot/Jupyter.scala

```diff
@@ -91,6 +93,95 @@ object Jupyter {
                             Some(xbounds))
     })
 
+  def whiskers[K, N](samples: Seq[Map[String, Double]]): Plot = {
+    val labels = samples.head.keys.toList.sorted
+    val data = labels.map { k =>
+      k -> samples.map { m =>
+        m(k)
+      }
+    }
+    whiskers(data, None)
+  }
+
+  def whiskers[K, N](seq: Seq[(K, Seq[N])], ybounds: Option[Bounds])(
+      implicit num: Numeric[N],
+      theme: Theme): Plot = {
+    val data = seq.map { case (_, ns)  => ns.map(num.toDouble) }
+    val labels = seq.map { case (k, _) => k.toString }
+
+    val xb = Bounds(0, seq.size.toDouble)
+    val yb = ybounds.getOrElse(
```

Comment:
These y bounds include the full range of data which we're not currently plotting anything for. Maybe this should use the BoxPlotSummaryStatistics to bound on the highest upperWhisker and lowest lowerWhisker?